lib/webpacker/railtie.rb

```diff
@@ -46,7 +46,9 @@ class Webpacker::Engine < ::Rails::Engine
   end
 
   initializer "webpacker.proxy" do |app|
-    if Webpacker.config.dev_server.present?
+    insert_middleware = Webpacker.config.dev_server.present? rescue nil
+    insert_middleware ||= false
+    if insert_middleware
```

Comment:
`nil` is equal to `false` in `if` statement
![image](https://user-images.githubusercontent.com/1024162/38550002-ee4c2820-3ce7-11e8-8486-05dd5bd49aac.png)

Reply:
you're right ! thanks 
ed6743a170242e7611795a21b9581b7992a6d061